www/app.py

```diff
@@ -3,10 +3,18 @@
 from os.path import isfile, join
 from flask_cors import CORS, cross_origin
 import xml.etree.ElementTree
+import Adafruit_GPIO.PWM as pwmLib
+import Adafruit_GPIO.GPIO as gpioLib
+import json
+import threading
```

Comment:
I think threading was just for the sensor polling, so I don't think you need it here.